src/governance/governance-object.cpp

```diff
@@ -624,6 +626,70 @@ int CGovernanceObject::CountMatchingVotes(vote_signal_enum_t eVoteSignalIn, vote
 *   Get specific vote counts for each outcome (funding, validity, etc)
 */
 
+int64_t CGovernanceObject::GetPaymentStartTime()
+{
+    UniValue data;
+    data.read(this->GetDataAsPlainString());
+    UniValue paymentStartData = find_value(data, "start_epoch");
+
+    if(paymentStartData.isNum())
+        return paymentStartData.get_int();
+
+    return -1;
+}
+
+int64_t CGovernanceObject::GetPaymentEndTime()
+{
+    UniValue data;
+    data.read(this->GetDataAsPlainString());
+    UniValue paymentEndData = find_value(data, "end_epoch");
+    if(paymentEndData.isNum())
```

Comment:
Please follow the [Coding Style](https://github.com/dashpay/dash/blob/master/doc/developer-notes.md#coding-style-c)